scripts/verify_tla_coverage.py

```
@@ -0,0 +1,132 @@
+coverage invariant. This harness runs each invariant in isolation and treats the
+expected witness violation as success.
+"""
+
+from __future__ import annotations
+
+import argparse
+import pathlib
+import re
+import subprocess
+import tempfile
+from typing import Sequence
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)